app/controllers/endpoints_controller.rb

```diff
@@ -4,8 +4,9 @@ class EndpointsController < ApplicationController
   before_action :find_endpoint, :only => %i[destroy show update]
 
   def index
-    records = paginate(policy_scope(Endpoint))
-    render :json => EndpointSerializer.new(records)
+    scope = policy_scope(Endpoint)
+    records = paginate(scope)
+    render :json => EndpointSerializer.new(records, pagination_options(scope))
```

Comment:
Can we refactor the pagination meta tag generation into base serializer?

``` ruby
class NotificationsSerializer
  include FastJsonapi::ObjectSerializer

  meta do |scope|
    {
      :total => scope.count, 
      :per_page => scope.per_page, 
      :page => scope.current_page
    } if scope.respond_to? :current_page
  end
end

Reply:
`scope` seems to be single instance being rendered. I'll try looking around if there's a way to get real scope here